src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/PackageIndexWriter.java

```diff
@@ -85,7 +85,7 @@ protected void addIndex(Content target) {
                 = configuration.group.groupPackages(packages);
 
         if (!groupPackageMap.keySet().isEmpty()) {
-            Table table =  new Table(HtmlStyle.summaryTable)
+            var table =  new Table<PackageElement>(HtmlStyle.summaryTable)
```

Comment:
Let's remove one redundant space
```suggestion
            var table = new Table<PackageElement>(HtmlStyle.summaryTable)
```